src/Catalyst.Abstractions/P2P/IPeerSettings.cs

```diff
@@ -30,9 +30,9 @@ namespace Catalyst.Abstractions.P2P
 {
     public interface IPeerSettings
     {
+        IPAddress PublicIpAddress { get; }	
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace all tab characters in this file by sequences of white-spaces.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4366149)